src/parsers/DE/DESlashDateFormatParser.js

```diff
@@ -0,0 +1,107 @@
+/*
+    Date format with slash "/" (also "-" and ".") between numbers
+    - Tuesday 11/3/2015
+    - 11/3/2015
+    - 11/3
+*/
+var moment = require('moment');
+var Parser = require('../parser').Parser;
+var ParsedResult = require('../../result').ParsedResult;
+
+var PATTERN = new RegExp('(\\W|^)' +
+    '(?:' +
+        '(?:am\\s*?)?' +
+        '((?:sonntag|so|montag|mo|dienstag|di|mittwoch|mi|donnerstag|do|freitag|fr|samstag|sa))' +
+        '\\s*\\,?\\s*' +
+        '(?:den\\s*)?' +
+    ')?' +
+    '([0-9]{1,2})[\\/\\.\\-]([0-9]{1,2})' +
```

Comment:
There is a bug here #143 there is a bugfix for it in PR #142 you can take a look there for the fix.